### DISCUSSION TITLE
File import/export location

### DISCUSSION THREAD

**Discussion by artvandelay750** [OpenBB-finance/OpenBB#1641] [Q&A]
Where is the default directory for the econometrics load?

2022 Apr 05, 15:05 (🦋) /econometrics/ $ load --file "/Users/artvandelay/Desktop/OPENBB-exports/20220405_145208_stocks_raw_data_AMC.xlsx" amc

I'm unable to specify an absolute path. There are a few different import/export folders, for --export, etf, portfolio import, etc

Thank you!
Art
acclaimedabs.com

**Comment by deeleeramone** [comment]
You are right, some of the --export functions have been left behind in the ability to specify the absolute path for them. They will still be in the /exports/ folder, but will be the folders for groups of commands - i.e. exports/stocks/raw_data

![Screen Shot 2022-06-07 at 9 37 03 AM](https://user-images.githubusercontent.com/85772166/172439358-673e6cdb-00ca-464d-abc4-5d47e23c831f.png)

**Comment by DidierRLopes** [comment]
Hey @artvandelay750,

The menu has changed to display where it is looking for files. As you can see from the screenshot that @deeleeramone shared above!

Hope this answers your question,
Didier
